.github/pull_request_template.md

```diff
@@ -0,0 +1,63 @@
+Fixes #
+
+Resolves #
```

Comment:
Do both `Fixes` and `Resolves` apply for the same issue number? If so, can we put them together like `Fixes/Resolves`? If not, could you distinguish them for my understanding?

Reply:
Both of them kind of does the same thing vai, from my point of view, I keep them as for following purpose.

Fixes #: This is used when the changes in the pull request directly address and resolve the issue mentioned in the linked ticket. 

Resolves #: This is used when the changes in the pull request indirectly resolve another issue mentioned in the linked ticket. It might not directly fix the problem, but it contributes to the resolution of that issue in some way.